src/github.rs

```diff
@@ -1745,11 +1755,60 @@ impl<'q> IssuesQuery for Query<'q> {
                         .get_comment(&client, fk_initiating_comment.try_into()?)
                         .await?;
 
+                    // To avoid constant (and counter-productive) pings we will only ping when
+                    //  - we are 2 weeks into the FCP
+                    //  - or when there are no concerns and we are at least 4 weeks into the FCP.
+                    //
+                    // FIXME: This should get T-compiler approval before being enabled by default
+                    let should_mention = std::env::var("TRIAGEBOT_COMPILER_MENTION").is_ok() && {
+                        let now = chrono::offset::Utc::now();
+                        let time_diff = now - init_comment.created_at;
+                        time_diff.num_weeks() == 2
+                            || (time_diff.num_weeks() >= 4 && fcp.concerns.is_empty())
+                    };
+
```

Comment:
```suggestion
                    // TODO: agree with the team(s) a policy to emit actual mentions to remind FCP
                    // voting member to cast their vote
                    let should_mention = false;
```

Reply:
@Urgau can we disable this for the moment and keep it as a reminder (which is useful)

Reply:
I applied your patch *(which removes, not disable, it was already disabled)*.

I hope we can get a policy on this in the near future as IMO not having mention doesn't give much benefits.

Reply:
yep, that's on my radar, don't worry :slightly_smiling_face: as I mentioned I just think that that needs a brief discussion involving people on the other end of these mentions and I don't want your work here to be blocked on that